package.json

```diff
@@ -73,17 +72,17 @@
     "yargs": "^3.4.5"
   },
   "devDependencies": {
-    "co-mocha": "^1.1.2",
-    "coveralls": "^2.11.4",
-    "eslint": "^1.7.3",
+    "coveralls": "^3.0.2",
+    "eslint": "^5.16.0",
     "husky": "^0.14.3",
     "istanbul": "^0.4.0",
     "lint-staged": "^5.0.0",
     "memdown": "^1.1.2",
-    "mocha": "^2.2.4",
-    "nock": "^1.6.0",
+    "mocha": "^6.1.4",
+    "nock": "^10.0.6",
+    "rxjs-compat": "^6.5.2",
```

Comment:
Otherwise, I was getting [this issue](https://github.com/okonet/lint-staged/issues/416)